### PR TITLE
Add makeup-expander definitions

### PR DESCRIPTION
**Pull request by darkwebdev**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/makeup-expander/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// Type definitions for makeup-expander 0.4
+// Project: https://github.com/makeup-js/makeup-expander
+// Definitions by: Timur Manyanov <https://github.com/darkwebdev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+declare class Expander {
+    constructor(el: HTMLElement, selectedOptions: any);
```
Comment (sheetalkamat) [inline types/makeup-expander/index.d.ts]: Need better typing for selected option as per https://github.com/makeup-js/makeup-expander#params

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/makeup-expander/index.d.ts (hunk 1 above)
Comment (darkwebdev) [reply types/makeup-expander/index.d.ts]: You're absolutely right, added options interface.

**Comment by nycdotnet** [comment]
it appears that you're changing a lot of files that are unrelated - for example reverting a recent PR.  Please confirm the changed files only includes the files you intended to change.  Thanks.  :-)

**Comment by darkwebdev** [comment]
@nycdotnet sorry, I was completely stupid to do that, fixed.

**Pull request merged**